packages/grafana-ui/src/components/Logs/LogRowMessage.tsx

```diff
@@ -124,18 +130,21 @@ class UnThemedLogRowMessage extends PureComponent<Props, State> {
           {row.searchWords && row.searchWords.length > 0 && (
             <span
               onClick={this.onContextToggle}
-              className={css`
-                visibility: hidden;
-                white-space: nowrap;
-                position: relative;
-                z-index: ${showContext ? 1 : 0};
-                cursor: pointer;
-                .${style.logsRow}:hover & {
-                  visibility: visible;
-                  margin-left: 10px;
-                  text-decoration: underline;
-                }
-              `}
+              className={cx(
+                styles.hoverColor,
+                css`
```

Comment:
Could we move this into getStyles (or getLogRowsStyles) function instead?

Reply:
Absolutely! Good point. Refactored.